cabal.project

```diff
@@ -210,7 +210,7 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-crypto/
-  tag: 3c707936ba0a665375acf5bd240dc4b6eaa6c0bc
+  tag: 2547ad1e80aeabca2899951601079408becbc92c
   --sha256: 0g8ln8k8wx4csdv92bz09pr7v9dp4lcyv1334b09c9rgwdwhqg1b
```

Comment:
Tip:
```
nix-prefetch-git https://github.com/input-output-hk/cardano-crypto 2547ad1e80aeabca2899951601079408becbc92c
Initialized empty Git repository in /tmp/git-checkout-tmp-mn30nsyu/cardano-crypto-2547ad1/.git/
remote: Enumerating objects: 104, done.
remote: Counting objects: 100% (104/104), done.
remote: Compressing objects: 100% (92/92), done.
remote: Total 104 (delta 5), reused 68 (delta 2), pack-reused 0
Receiving objects: 100% (104/104), 152.66 KiB | 873.00 KiB/s, done.
Resolving deltas: 100% (5/5), done.
From https://github.com/input-output-hk/cardano-crypto
 * branch            HEAD       -> FETCH_HEAD
Switched to a new branch 'fetchgit'
removing `.git'...

git revision is 2547ad1e80aeabca2899951601079408becbc92c
path is /nix/store/bvxsj0yv59b5k3yzsgbl2082i97lc5z8-cardano-crypto-2547ad1
git human-readable version is -- none --
Commit date is 2020-02-18 09:14:29 +0800
hash is 1p2kg2w02q5w1cvqzhfhqmxviy4xrzada3mmb096j2n6hfr20kri
{
  "url": "https://github.com/input-output-hk/cardano-crypto",
  "rev": "2547ad1e80aeabca2899951601079408becbc92c",
  "date": "2020-02-18T09:14:29+08:00",
  "sha256": "1p2kg2w02q5w1cvqzhfhqmxviy4xrzada3mmb096j2n6hfr20kri",
  "fetchSubmodules": false
}
```

So `1p2kg2w02q5w1cvqzhfhqmxviy4xrzada3mmb096j2n6hfr20kri` is the right value here

Reply:
Right, I should stop abusing ci to give me the correct hash :)